pkg/datapath/linux/arp/ping.go

```diff
@@ -0,0 +1,108 @@
+package arp
```

Comment:
I notice some similarities between this file and the original source, which is licensed MIT with a particular copyright owner. If the code is being adopted from the external project with that license, then these submissions must respect those conditions, specifically retaining the copyright and license statement.

Reply:
Thanks for providing the feedback! I'm still working on this and if the project would be ok to accept the code as a copy from the original, I will definitely add the attribution. For now, my goal was to do a blind copy  to get a quick understanding on what would be required to be re-implemented.

Reply:
Is this resolved?

Reply:
@vladdy I'm referring to the comment that Joe left regarding the copyright.

Reply:
The code as it stands holds a copyright of 2018-2020 which suggests that the code is indeed derivative and therefore must adhere to the conditions outlined in the original source. If so, this is not currently resolved and will need to be addressed.

If this really is brand new code then I would suggest bumping the copyright years to reflect that.

Reply:
I'll bump the version, as I did not know what are the rules about that in the project and just copied the header for an existing file.

Reply:
I've updated the year in the copyright. Let me know if anything else is needed with regards to this matter.

Reply:
@joestringer ping! Do you think this conversation can be resolved?

Reply:
Yup, thanks!